plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -0,0 +1,208 @@
+                <Text weight={700} size={48}>
+                  ようこそ
+                </Text>
+              </TextWrapper>
+              <ImgSection>
+                <TextWrapper width={215} height={32}>
+                  <Text weight={500} size={20}>
+                    マップを使ってみる
+                  </Text>
+                </TextWrapper>
+                <ImgWrapper>
+                  <img src={welcomeScreenVideo} onClick={handleShowVideo} />
+                </ImgWrapper>
+              </ImgSection>
+            </TryMapWrapper>
+            <BtnsWrapper>
+              <ButtonWrapper>
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+              <ButtonWrapper>
+                <Icon size={20} icon="plusCircle" color="#fafafa" />
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+            </BtnsWrapper>
+          </InnerWrapper>
+          <CheckWrapper>
+            <Checkbox checked={dontShowAgain} onClick={handleDontShowAgain}>
+              <TextWrapper width={192} height={46}>
+                <Text weight={700} size={14}>
+                  閉じて今後は表示しない
+                </Text>
+              </TextWrapper>
+            </Checkbox>
+          </CheckWrapper>
+        </>
+      ) : (
+        <>
+          <CloseButton>
+            <Icon size={32} icon="close" onClick={handleCloseVideo} />
+          </CloseButton>
+          <VideoWrapper>
+            <Video width=" 1142" height="543" src="https://www.youtube.com/embed/pY2dM-eG5mA" />
+          </VideoWrapper>
+        </>
+      )}
+    </Wrapper>
+  );
+};
+export default WelcomeScreen;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  gap: 50px;
+  background: rgba(0, 0, 0, 0.7);
+`;
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  padding: 0px;
+  gap: 119px;
+  width: 742px;
+  height: 316px;
+  left: calc(50% - 742px / 2 + 0.5px);
+  top: calc(50% - 316px / 2 - 30.5px);
+`;
+const TextWrapper = styled.div<{ width: number; height: number }>`
+  width: ${({ width }) => width}px;
+  height: ${({ height }) => height}px;
+  margin: 0px;
+`;
+
+const Text = styled.p<{ weight: number; size: number }>`
+  font-weight: ${({ weight }) => weight}px;
+  font-size: ${({ size }) => size}px;
+  margin-bottom: 0px;
+  color: #fafafa;
+`;
+
+const TryMapWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-end;
+  padding: 0px;
+  gap: 55px;
```

Comment:
I think you are using gap here, in BtnsWrapper, etc almost like padding, no? Gap should be used to create a gap between the children inside a flex container. If I am right, please switch from gap to padding in each of these components instead.

Reply:
it is to set the gap between the inner wrapper and the checkbox wrapper

Reply:
I replaced it with padding top for the checkbox wrapper

Reply:
a36d7887f9fae43211d25e5df9545ce613bbccec